exonum/src/runtime/dispatcher/mod.rs

```diff
@@ -360,6 +365,17 @@ impl Dispatcher {
         );
         Ok(())
     }
+
+    /// Checks if provided instance ID is greater than the currently stored one
+    /// and updates it if needed.
+    fn update_max_builtin_id(&self, fork: &Fork, instance_id: InstanceId) {
```

Comment:
Don't we need to return `Restult<(), Error>` here and return `Err(_)` if `current_max_id` is `None` ?

Reply:
Nope. `None` means that field was not initialized. Here we use that `Option` implements `PartialEq`: `Some(val)` is always greater than `None`, so at the first iteration (when field is `None`) it will be initialized correctly.